### DISCUSSION TITLE
Dexcom and xDrip running together simultaneously

### DISCUSSION THREAD

**Discussion by joshg1994** [NightscoutFoundation/xDrip#3316] [Q&A]
Is it recommended to have xdrip and dexcom run together simultaneously? 

I was noticing a lot of significant inaccuracies with blood sugar readings while using xdrip alone, and comparing to finger pokes. I wanted to compare with my dexcom readings, and noticed dexcom was more accurate. But after a little bit of time running together simultaneously, dexcom and xdrip is identical. Not sure if this is just a coincidence, or something else. 

At first I was under the impression that you can't have them running together simultaneously.

**Comment by Navid200** [comment]
Dexcom is a manufacturer, not a sensor.
What Dexcom sensor are you asking about?

**Comment by joshg1994** [comment]
I am on the dexcom g7

**Reply by Navid200** [reply]
If you need to upload to Clarity, with G7, you can use the Dexcom receiver.  If you get a receiver, you only need to have it near you once a day to backfill all readings of the past 24 hours.
Then, once a month, you can use the receiver to upload to Clarity.

If you don't need to upload to Clarity, you can use xDrip alone.

If you want to use Dexcom app and xDrip together for G7, you have to use Companion app mode.  

It's up to you and only you know what you need.

If you have any questions, ask.

**Reply by Navid200** [reply]
And xDrip and Dexcom app report exactly the same readings.  G7 does not send raw values.  Therefore, xDrip cannot present anything to you other than the readings sent by the transmitter, which the Dexcom app would report as well.
